templates/homepage-community-band.tmpl

```diff
@@ -19,8 +18,6 @@
         {% endfor %}
       </div>
     </div>
-    <hr />
+    {% include 'homepage-back-to-top.tmpl' %}
   </div>
 </div>
-{% include 'homepage-bullhorn-band.tmpl' %}
```

Comment:
We only just merged these two items together in #116, is there a reason we're splitting it out again? I agreed with your reasoning in that PR, but now it feels like we're flipflopping

Reply:
@GregSutcliffe Yeah, probably my bad and merging those two items was likely premature as it deviates from the wireframe, even though I feel like my logic still stands. I've reverted back so we can just make the homepage as close as possible to the wireframe. Perhaps we'll flipflop again but that should happen with community feedback on the wireframe that Tina has designed.

Reply:
Alright, that's strong enough. No more deviations! :grin:

Reply:
Sound. I took out some of the back to top links that I added in as well. There are one or two slight deviations to the wireframe (like the "view more" link in the Contribute section and the "Events" link in the tab menu bar) but I really don't think we should revert them.

Reply:
@oraNod I just want to add that I think your logic made sense in terms of the order but I think I just hadn't updated my wireframe to match that yet so sorry about that! If it's an easy fix, definitely feel free to adjust but also can make more adjustments as we get feedback from the community! 😄

Reply:
Now we're definitely flipflopping :)

Reply:
Sorry! Just wanted to state my agreement since I didn't get a chance to but definitely don't need to do anything with it now.

Reply:
No worries, @tiyiprh A bit of flipflopping never hurt and it's better to take those kind of decisions through the right process. It would be much better to make that - and any other changes that differ from the wireframe - with your input while giving the community a chance to look at it.